frontend/src/app/features/work-packages/components/wp-card-view/wp-single-card/wp-single-card.component.ts

```diff
@@ -116,7 +137,7 @@ export class WorkPackageSingleCardComponent extends UntilDestroyedMixin implemen
     const base = 'op-wp-single-card';
 
     return {
-      [`${base}_checked`]: this.isSelected(this.workPackage),
+      [`${base}_checked`]: this.selected,
```

Comment:
I would align the `checked` and `selected` naming here if they're the same thing